autoPyTorch/ensemble/ensemble_builder.py

```diff
@@ -1033,20 +1027,19 @@ def get_n_best_preds(self) -> List[str]:
         num_keys = len(sorted_keys)
         # remove all that are at most as good as random
         # note: dummy model must have run_id=1 (there is no run_id=0)
-        dummy_scores = list(filter(lambda x: x[2] == 1, sorted_keys))
-
+        dummy_losses = list(filter(lambda x: x[2] == 1, sorted_keys))
         # Leave this here for when we enable dummy classifier/scorer
-        if dummy_scores:
+        if dummy_losses:
             # number of dummy models
-            num_dummy = len(dummy_scores)
-            dummy_score = dummy_scores[0]
-            self.logger.debug("Use %f as dummy score" % dummy_score[1])
-            sorted_keys = list(filter(lambda x: x[1] > dummy_score[1], sorted_keys))
+            num_dummy = len(dummy_losses)
+            dummy_loss = dummy_losses[0]
+            self.logger.debug("Use %f as dummy loss" % dummy_loss[1])
+            sorted_keys = list(filter(lambda x: x[1] < dummy_loss[1], sorted_keys))
 
             # remove Dummy Classifier
             sorted_keys = list(filter(lambda x: x[2] > 1, sorted_keys))
             if not sorted_keys:
```

Comment:
same here

Reply:
Thanks,  I just pushed a new commit with this.